src/components/Layout/tests/Layout.test.tsx

```diff
@@ -30,32 +30,6 @@ describe('<Layout />', () => {
     expect(section.find(MyComponent).exists()).toBe(true);
   });
 
-  describe('<Layout.Section />', () => {
```

Comment:
We shouldn't expect on the base components props since it doesn't provide value. These tests should be handled by percy